source/features/index.tsx

```diff
@@ -211,12 +211,15 @@ function enforceDefaults(
 	}
 }
 
+const getFeatureID = (url: string): FeatureID => url.split('/').pop()!.split('.')[0];
+
 /** Register a new feature */
-const add = async (id: FeatureID, ...loaders: FeatureLoader[]): Promise<void> => {
+const add = async (url: string, ...loaders: FeatureLoader[]): Promise<void> => {
+	const id = getFeatureID(url);
 	/* Feature filtering and running */
 	const options = await globalReady;
 	// Skip disabled features, unless the "feature" is the fake feature in this file
-	if (!options[`feature:${id}`] && id as string !== __filebasename) {
+	if (!options[`feature:${id}`] && id !== getFeatureID(import.meta.url)) {
```

Comment:
3/2

We can probably also use the `rgh-` check here 
```suggestion
	if (!options[`feature:${id}`] && !id.startsWith('rgh')) {
```